packages/picker/src/picker.css

```diff
@@ -36,6 +36,39 @@ governing permissions and limitations under the License.
     min-width: 0;
 }
 
+:host([size]) {
+    /* TODO: https://github.com/adobe/spectrum-css/blob/011ca5408ae5eb32bb55e1cf79c06e40171fd17c/components/picker/index.css#L93 */
+    --spectrum-picker-width: var(--spectrum-global-dimension-size-2400);
+}
+
+:host([size='s']) sp-progress-circle {
+    --spectrum-progress-circle-size: var(--spectrum-workflow-icon-size-75);
+    --spectrum-progress-circle-thickness: var(
+        --spectrum-progress-circle-thickness-small
+    );
+}
+
+:host([size='m']) sp-progress-circle {
+    --spectrum-progress-circle-size: var(--spectrum-workflow-icon-size-100);
+    --spectrum-progress-circle-thickness: var(
+        --spectrum-progress-circle-thickness-small
+    );
+}
+
+:host([size='l']) sp-progress-circle {
+    --spectrum-progress-circle-size: var(--spectrum-workflow-icon-size-200);
+    --spectrum-progress-circle-thickness: var(
+        --spectrum-progress-circle-thickness-medium
+    );
+}
+
+:host([size='xl']) sp-progress-circle {
+    --spectrum-progress-circle-size: var(--spectrum-workflow-icon-size-300);
+    --spectrum-progress-circle-thickness: var(
+        --spectrum-progress-circle-thickness-large
+    );
+}
```

Comment:
These values appear slightly different than those in Button, please make sure they align. We could even go so far as to externalize these styles to Progress Circle and leverage them from there in both places if we wanted.

Reply:
Yep, these were here since the PR was first opened, I was not sure if you guys already discussed around them.
Wouldn't this `--spectrum-progress-circle-size` make more sense in [spectrum-css](https://github.com/adobe/spectrum-css/blob/main/components/picker/index.css#L88)? I observed that in the spectrum-css storybook the spinner has the same dimension regardless of the Picker's size. Pinging again @pfulton about this 😄